release_notes.md

```diff
@@ -3,6 +3,8 @@
 - My change description (#PR)
 -->
 
+* Add host.json payload (Concurrency and Extensions) to the SyncTriggerPayload (ArmCacheEnabled = true) (#7863)
```

Comment:
Let's reword this - we're not adding host.json. We're adding a few new select configuration sections

Reply:
Re-written it. Could you have a look?